en/docs/observability/perform-root-cause-analysis.md

```diff
@@ -1,96 +1,81 @@
 
-!!! info
-    For explanation, we are using the sample data available in the product.
-    
-## Get started with the sample service
+This guide describes how you can detect anomalies and perform root cause analysis on a service deployed in Choreo. For demonstration, let’s use the observability sample service available in Choreo.
 
-1. Go to **Services** from the left navigation.
-2. Create a new service.
-    1. **If this is your first service**, 
     
-        1. Enter a name for the service and click **Create**. 
-           
-           ![Create a new service](../assets/img/observability/first-service-creation.png){.cInlineImage-full}
-           
-    2. **Else** (if you already have services created), 
-        
-        1. Click **+ Create** to start creating a new service.
-        2. Enter a name for the service and click **Create**.
-           
-           ![Create a new service](../assets/img/observability/service-creation.png){.cInlineImage-full}
-
-3. From the left-hand menu, click **Observe**.
-4. On the right-hand bottom, click the **Sample Service** icon.
-   
-    ![Sample service popup](../assets/img/observability/sample-service-pop-up.png){.cInlineImage-full}
-   
-5. Click **Try Sample Service**.
-
-    ![Try sample service](../assets/img/observability/try-sample-service.png){.cInlineImage-half}
+## Step 1: Get started with the sample service
+Follow this procedure to try out the sample service:
+
+1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/).
+2. Go to the **Observability** card and click **Try Now**. This takes you to the **Observability** page. 
+3. Click **Try Sample**. You are directed to the sample service low-code view where you can see the following:
+    - Low-code diagram
+    - Throughput graph   
+    - Latency graph
+    - Logs
+    - Diagnostic data
+
+Now you are ready to detect anomalies and perform root cause analysis on the sample service.
     
-## Detect anomalies and perform root cause analysis
-You can detect the anomalies of the usage of a service by observing the **Throughput & Latency** graphs. The throughput graph shows the successful and erroneous requests that occurred during a period. The latency graph shows the latencies of each request. You can observe these graphs at a lower granularity by choosing a shorter time range, which gives you a better understanding of the incidents.
-
-### Step 1: Detect the anomalies
+## Step 2: Detect anomalies and perform root cause analysis
 
-#### Analyze the throughput graph
-The throughput graph shows the throughput of requests per hour for a selected timestamp. 
+To detect anomalies of the service, you must trace its executions from the code statement level to the machine it runs in. You can analyze observability data and logs to troubleshoot and identify the root cause for the anomalies you detect. 
```

Comment:
you must trace its executions from the code statement level to the machine it runs in -> this statement is not very clear

Reply:
How about "Anomalies may occur for a service due to errors that occur at the code statement level or at the CPU level."
@Nashaath, can you please confirm whether this gives the same idea that you originally intended to give?

Reply:
I went through the product page and wanted to make sure docs also give the same idea when it comes to observability. So thought it would be appropriate to mention this because Choreo supports this level of tracing. 
<img width="1430" alt="Screenshot 2021-06-17 at 07 59 16" src="https://user-images.githubusercontent.com/9528121/122322241-cc75de80-cf42-11eb-8d59-102a02b71e3e.png">
+1 to rephrase as suggested.

Reply:
Changed in https://github.com/wso2/docs-choreo-dev/pull/182